src/catchup/CatchupManagerImpl.cpp

```diff
@@ -137,10 +137,6 @@ CatchupManagerImpl::processLedger(LedgerCloseData const& ledgerData)
                               "ledger {} was closed on network",
                               lastLedgerInBuffer);
 
-        // We only need ledgers starting from the checkpoint. We can
-        // remove all ledgers before this
-        mSyncingLedgers.erase(mSyncingLedgers.begin(), it);
```

Comment:
what about simplifications on line 128? We don't need to search for the first checkpoint ledger anymore. I think we can just call isFirstLedgerInCheckpoint() on the first ledger of mSyncingLedgers

Reply:
Makes sense! I updated it.